plugins/modules/net_tools/nmcli.py

```diff
@@ -106,6 +106,11 @@
             - A list of DNS search domains.
         elements: str
         type: list
+    method4:
+        description:
+            - Configuration method to be used for ipv4
```

Comment:
```suggestion
            - Configuration method to be used for IPv4.
```
Please also mention the relation to the `ip4` option (also mention this option in the documentation of the `ip4` option). And the same for `ip6`/`method6` :)

Reply:
Ok ! Thanks a lot for the review !
I will do that tomorrow. This is midnight here, I will probably generate more issues if I continue to work today 😊